fsharp-backend/src/Wasm/Wasm.fs

```diff
@@ -310,10 +310,13 @@ type EvalWorker =
           )
         with
         | e ->
```

Comment:
The 'top' case here handles parsing, and the below one handles exceptions running analysis.